bin/cml/report.js

```diff
@@ -0,0 +1,8 @@
+exports.command = 'report';
```

Comment:
some options:

1. `cml {report,check} create`: (current) doesn't reflect it's really a comment/check
2. `cml {comment,check} create`: doesn't map well with product description ("CML does **runner**s (`runner` :heavy_check_mark:) and **report**ing (`{comment,check}` :question:)
3. `cml report create [{--comment,--check}]`: a bit long
4. `cml report {comment,check} [--create]`: clunky

Reply:
Not to drag this out more, I agree. `cml report` feels off; cml {comments} a report you make. it doesn't make a report for you.

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/f01975ab39b0dca904f636bb8edc9d1aac37c3e1 by renaming `cml report create` to `cml comment create`

Reply:
If we plan to provide a `cml report create` with some higher-level abstraction over comments and checks, it belongs to a separate pull request.